lib/crowbar/client/request/services.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2017, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  module Client
+    module Request
+      #
+      # Module for the List Restarts request implementations
+      #
+      module Services
+        autoload :ListRestarts, File.expand_path("../services/list_restarts", __FILE__)
+        autoload :ClearRestart, File.expand_path("../services/clear_restart", __FILE__)
+        autoload :RestartFlags, File.expand_path("../services/restart_flags", __FILE__)
+        autoload :DisableRestart, File.expand_path("../services/disable_restart", __FILE__)
```

Comment:
The only small comment I have is that it's not 100% clear from the names here which item is working on the the databag (to configure which cookbook is using this feature) and which item is working on the nodes (to find out which service on which node needs a restart). But maybe it's just me.

Reply:
Ummm, I just followed the naming convention of other commands here basically. This is the client so this only does simple requests, so Im not sure you should know which item is working on the databag from here, for that you should check on the crowbar_framework.

I can change them thougth, albeit you are risking me setting some terrible names here :P

Reply:
oooh, I get it. damn, I need more coffe